### PR TITLE
ACP uses loginhandler properly

### LINKED ISSUE #1757: ACP should use loginhandler properly
Update Admin CP login to use the validate_login from the login datahandler to provide a consistent format for user verification.

http://community.mybb.com/thread-165507.html

### PR DESCRIPTION
**Pull request by tommm**
Fixes #1757

### CODE REVIEW

File: inc/datahandlers/login.php [hunk 1]
```diff
@@ -237,7 +237,10 @@ function validate_login()
 
 		$plugins->run_hooks('datahandler_login_validate_start', $this);
 
-		$this->verify_attempts($mybb->settings['captchaimage']);
+		if(!defined(‘IN_ADMINCP’))
```
Comment (ATofighi) [inline inc/datahandlers/login.php]: it should be
```
if(!defined('IN_ADMINCP'))
```

Comment (tommm) [inline inc/datahandlers/login.php]: How strange! Changing.

**Comment by Cu8eR** [comment]
Looks good for me

**Comment by PaulBender** [comment]
Looks good to me as well

**Pull request merged**